src/xGadgetron/pGadgetron/tests/tests_mrimage_geometry.py

```diff
@@ -0,0 +1,63 @@
+
+# -*- coding: utf-8 -*-
+"""sirf.Gadgetron test.
+v{version}
+
+Constructor of MR image data from MR acquisition data
```

Comment:
- the name of the file implies it's testing something else. Maybe the file should be renamed?
- add some more info on what this "test" is actually doing. Would be useful to mention it's writing a DICOM file for instance...
- are there any actual tests that could be done? (I guess only appropriate if we know what the input is for sure is going to be)

Reply:
let's just remove this file

Reply:
done